### PR TITLE
Improvement: normalise shared code

### PR DESCRIPTION
**Pull request by melowe**
Move repeated functional interfaces and callback execute functions into shared location.

### CODE REVIEW

File: tests/jmeter-test/src/test/java/com/quorum/tessera/jmeter/PlaceHolder.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package com.quorum.tessera.jmeter;
```
Comment (SatpalSandhu61) [inline tests/jmeter-test/src/test/java/com/quorum/tessera/jmeter/PlaceHolder.java]: Is this meant to be part of this PR?

File: config/217391d4-bdd1-4bd3-9364-1335d9522ba9.key [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (SatpalSandhu61) [inline config/217391d4-bdd1-4bd3-9364-1335d9522ba9.key:1]: Also not sure why these keys are being added. They don't appear to be used anywhere.

File: tests/jmeter-test/src/test/java/com/quorum/tessera/jmeter/PlaceHolder.java (hunk 1 above)
Comment (melowe) [reply tests/jmeter-test/src/test/java/com/quorum/tessera/jmeter/PlaceHolder.java]: Yes. That is the fix.

Comment (SatpalSandhu61) [reply tests/jmeter-test/src/test/java/com/quorum/tessera/jmeter/PlaceHolder.java]: But this PR isn't for the jmeter fix - it's only meant to be a refactoring of the code into a shared module isn't it?

Comment (melowe) [reply tests/jmeter-test/src/test/java/com/quorum/tessera/jmeter/PlaceHolder.java]: I see what you mean

**Pull request closed without merging**